db/migrate/20210107175100_add_virtual_to_diaper_drives.rb

```diff
@@ -0,0 +1,8 @@
+# Organizations can remind Partners when their deadlines are for putting in their requests
+class AddVirtualToDiaperDrives < ActiveRecord::Migration[5.2]
+  def change
+    change_table :diaper_drives do |t|
+      t.boolean :virtual
```

Comment:
```suggestion
      t.boolean :virtual, default: false, null: false
```
Let's backfill every record as `false` and make sure that the value can either be `true` or `false` and not `nil`. This can make building SQL queries a bit easier in the future (outside of Rails).